catalog-info.yaml

```
@@ -2,11 +2,12 @@ apiVersion: backstage.io/v1alpha1
 kind: Component
 metadata:
   name: "mobile-developer-guide"
+  title: "Mobile development guide"
   description: "A guide to mobile application development for the Government of the Province of British Columbia"
   annotations:
     github.com/project-slug: bcgov/mobile-developer-guide
     backstage.io/techdocs-ref: dir:.
 spec:
   type: documentation
   lifecycle: experimental
```

```suggestion
  lifecycle: production
```
I know this wasn't part of your changes. But since we're in here I thought we could change the lifecycle from experimental to production. I had set it to experimental when I was first setting this up. I think it could change to production now.